src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
```

Comment:
according to the task, only services should be tested, but I think you can leave the tests for FruitTransaction class if the mentors are not opposed